### PR TITLE
Add overrides for fluentbit

### PR DESCRIPTION
**Pull request by rakesh-garimella**
**Description**

Changes proposed in this pull request:

- More overrides for fluentbit.

**Related issue(s)**

#8055 #8074

### CODE REVIEW

File: resources/logging/charts/fluentbit/templates/configmap.yaml [hunk 1]
```diff
@@ -33,6 +33,9 @@ data:
 {{- if .Values.conf.Output.Elasticsearch.enabled }}
     @INCLUDE output-elasticsearch.conf
 {{- end }}
+{{- if .Values.conf.Filter.record_modifier.cluster_name }}
```
Comment (a-thaler) [inline resources/logging/charts/fluentbit/templates/configmap.yaml:36]:
```suggestion
{{- if .Values.conf.Filter.record_modifier.enabled }}
```

File: resources/logging/charts/fluentbit/values.yaml [hunk 2]
```diff
@@ -166,11 +166,18 @@ conf:
...
       # When Merge_Log is enabled, the filter tries to assume the log field from the incoming message is a JSON string message and make a structured representation of it at the same level of the log field in the map. Now if Merge_Log_Key is set (a string name), all the new structured fields taken from the original log content are inserted under the new key.
       Merge_Log_Key: "logs"
     custom_parser:
-      enable: false
+      enabled: false
       name:
+    record_modifier:
+      enabled: false
+      Match: "*"
+      Key: cluster_name
```
Comment (a-thaler) [inline resources/logging/charts/fluentbit/values.yaml:179]: I think we should keep it generic, please put something like "myKey" and "myValue" as default values

Comment (rakesh-garimella) [reply resources/logging/charts/fluentbit/values.yaml]: fixed.

**Pull request merged**